docs/database_migration.md

```diff
@@ -102,4 +104,9 @@ or
 
 Modify your `database_url` in `lldap_config.toml` (or `LLDAP_DATABASE_URL` in the env)
 to point to your new database (the same value used when generating schema). Restart
-LLDAP and check the logs to ensure there were no errors.
\ No newline at end of file
+LLDAP and check the logs to ensure there were no errors.
+
+### More details/examples can be seen in the CI process [here](https://raw.githubusercontent.com/nitnelave/lldap/main/.github/workflows/docker-build-static.yml)
+* Postgres L308-327
```

Comment:
I'm wary of putting line numbers, since any diff in the workflow will lead to outdated line numbers. Can you put maybe job names?